File: java/projects/Drink_Order.java
import java.util.Scanner;

public class Drink_Order {

	public static void main(String[] args) {

		Scanner input = new Scanner(System.in);

		System.out.println("What type of drink would you like to order?");
		System.out.println("1. Water\n2. Coffee\n3. Tea");
		System.out.print("Drink selection #: ");

		// String variable to hold drink details
		String drinkDetails = "No drink chosen.";

		int choice = input.nextInt();

		// Remove newline left in input
		input.nextLine();

		// ---------------- WATER ----------------
		if (choice == 1) {

			drinkDetails = "Water";

			System.out.println("Would you like that 1) hot or 2) cold?");
			System.out.print("Enter temperature selection #: ");

			choice = input.nextInt();
			input.nextLine();

			if (choice == 1) {

				drinkDetails += ", hot";

			}
			else if (choice == 2) {

				drinkDetails += ", cold";

				System.out.print("Would you like ice? (Y/N) ");

				String response = input.nextLine();

				char yesNo = response.charAt(0);

				if (yesNo == 'Y' || yesNo == 'y') {
					drinkDetails += ", with ice";
				}
			}
			else {
				System.out.println("Not a valid temperature selection.");
			}
		}

		// ---------------- COFFEE ----------------
		else if (choice == 2) {

			drinkDetails = "Coffee";

			System.out.print("Would you like decaf? (Y/N): ");

			String decafResponse = input.nextLine();
			char decafYesNo = decafResponse.charAt(0);

			if (decafYesNo == 'Y' || decafYesNo == 'y') {
				drinkDetails += ", decaf";
			}

			System.out.println(
			    "Would you like 1) milk, 2) cream, or 3) none?"
			);

			System.out.print("Enter choice #: ");

			int milkCreamChoice = input.nextInt();
			input.nextLine();

			if (milkCreamChoice == 1) {
				drinkDetails += ", milk";
			}
			else if (milkCreamChoice == 2) {
				drinkDetails += ", cream";
			}

			System.out.print("Would you like sugar? (Y/N): ");

			String sugarResponse = input.nextLine();
			char sugar = sugarResponse.charAt(0);

			if (sugar == 'Y' || sugar == 'y') {
				drinkDetails += ", sugar";
			}
		}

		// ---------------- TEA ----------------
		else if (choice == 3) {

			drinkDetails = "Tea";

			System.out.print(
			    "Type of tea: 1) Black or 2) Green: "
			);

			int teaChoice = input.nextInt();
			input.nextLine();

			if (teaChoice == 1) {
				drinkDetails += ", black";
			}
			else if (teaChoice == 2) {
				drinkDetails += ", green";
			}
			else {

				// Invalid selection - assume black tea
				drinkDetails += ", black";

				System.out.println(
				    "Not a valid choice. Assuming black tea."
				);
			}
		}

		// ---------------- INVALID DRINK ----------------
		else {

			System.out.println(
			    "Sorry, not a valid drink selection."
			);
		}

		// Print final order
		System.out.println(
		    "Your drink selection: " + drinkDetails + "."
		);
	}
}

File: python/projects/drink_order.py
#we are creating a drink order prgram where users
#will be first prompted with the choice of water, coffe or tea.
#Based on those selections there are additional prompts that are unique to each.
#As the user enters each options, a string will be built 
#to indicate the entire order to be output at the end. 

#Would you like to order a drink today? y or n 
#Would you like water? if yes offer hot or cold , if cold offer ice or no ice
#if no, offer coffee etc...

# here is the actual program 
drinkDetails=""
drink = input("What type of drink would you like to order today? \nWater\nCoffee\nTea\nEnter your choice: ")
if drink == "Water":
    drinkDetails=drink
    temperature = input("Would you like your water? Hot or Cold:  ")
    if temperature == "Hot":
        drinkDetails += " , " + temperature
    elif temperature == "Cold":
        drinkDetails += " , " + temperature
    else:
        drinkDetails += " , wrong temperature entered" 
    Ice = input("Would you like Ice with your water? Yes or No : ")
    if Ice == "Yes":
        drinkDetails += ", Ice "
    else:
        drinkDetails += " , unknown temperature entered."
elif drink == "Coffee":
    drinkDetails=drink 
    coffeeOptions = input(" How would you like you coffee? Cuban or Colombian : ")
    if coffeeOptions ==" Cuban":
        drinkDetails +=" , " + coffeeOptions
    elif coffeeOptions =="Colombian" :
        drinkDetails += " , " + coffeeOptions 
    else:
        drinkDetails += " , sorry, we dont have your coffee selcetion available, try again"
elif drink =="Tea":
    drinkDetails=drink
    teaOptions = input("How would you like your tea? Green or Black? :  ")
    if teaOptions == "Green":
        drinkDetails +=" , " + teaOptions
    elif teaOptions == "Black": 
        drinkDetails += " , " + teaOptions
    else: 
        drinkDetails += ", sorry, we don't have that tea option available, try again"
else:
    print("Sorry, we did not have that drink available in our menu selection.")
print("Your drink selection: ", drinkDetails)


